test/cmd/policy.sh

```diff
@@ -26,7 +26,7 @@ os::cmd::expect_success 'oc new-project foo'
 os::cmd::expect_failure 'oc whoami --as=system:admin'
 os::cmd::expect_success_and_text 'oc whoami --as=system:serviceaccount:foo:default' "system:serviceaccount:foo:default"
 os::cmd::expect_failure 'oc whoami --as=system:serviceaccount:another:default'
-os::cmd::expect_success 'oc login -u system:admin -n cmd-policy'
+os::cmd::expect_success "oc login -u system:admin -n '$( oc project -q )'"
```

Comment:
the point of this was to switch out of the foo project back to the cmd-policy project. this will leave you in the foo project, which is being deleted, and the local role grants below will fail